.concourse/pipeline.yaml.gomplate

```diff
@@ -179,51 +182,109 @@ resources:
     regexp: kubecf-bundle-v(.*).tgz
 
 deploy_args: &deploy_args
-- -xce
+- -ce
 - |
+
+  # Login to gcloud
+  printf "%s" '((gke-suse-cap-json))' > $PWD/gke-key.json
+  export GKE_CRED_JSON=$PWD/gke-key.json
+  gcloud auth activate-service-account --key-file $PWD/gke-key.json
+
+  export GKE_PROJECT={{ if has . "gke_project" }}{{ .gke_project }}{{ else }}"suse-225215"{{ end }}
+  export GKE_CLUSTER_ZONE={{ if has . "gke_zone" }}{{ .gke_zone }}{{ else }}"europe-west3-c"{{ end }}
+  export GKE_CLUSTER_NAME="kubecf-ci-$(cat semver.gke-cluster/version | sed 's/\./-/g')"
+
+  gcloud --quiet beta container \
+    --project "${GKE_PROJECT}" clusters create "${GKE_CLUSTER_NAME}" \
+    --zone "${GKE_CLUSTER_ZONE}" \
+    --no-enable-basic-auth \
+    --machine-type "n1-highcpu-16" \
+    --image-type "UBUNTU" \
+    --disk-type "pd-ssd" \
+    --disk-size "100" \
+    --metadata disable-legacy-endpoints=true,build-url="${ATC_EXTERNAL_URL}" \
+    --scopes "https://www.googleapis.com/auth/devstorage.read_only","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly","https://www.googleapis.com/auth/trace.append" \
+    --preemptible \
+    --num-nodes "1" \
+    --enable-stackdriver-kubernetes \
+    --enable-ip-alias \
+    --network "projects/${GKE_PROJECT}/global/networks/default" \
+    --subnetwork "projects/${GKE_PROJECT}/regions/$(echo ${GKE_CLUSTER_ZONE} | sed 's/-.$//')/subnetworks/default" \
+    --default-max-pods-per-node "110" \
+    --no-enable-master-authorized-networks \
+    --addons HorizontalPodAutoscaling,HttpLoadBalancing \
+    --no-enable-autorepair
+
+  # Get a kubeconfig
+  gcloud container clusters get-credentials ${GKE_CLUSTER_NAME} --zone ${GKE_CLUSTER_ZONE} --project "${GKE_PROJECT}"
+
   export SCF_LOCAL="${PWD}/kubecf"
   export SCF_CHART="$(readlink -f s3.kubecf-ci/*.tgz)"
   export SCF_OPERATOR=true
   export FORCE_DELETE=true
   export HELM_VERSION="v3.1.1"
   export SCF_TESTGROUP=true
-  export BACKEND=imported
+  export BACKEND=gke
   export DOCKER_ORG=cap-staging
   export QUIET_OUTPUT=true
   export DOWNLOAD_CATAPULT_DEPS=false
-  export CLUSTER_NAME="$(cat kind-environments/name)"
-  export KUBECFG="$(readlink -f kind-environments/metadata)"
+  export KUBECFG="$(readlink -f ~/.kube/config)"
   pushd catapult
   # Bring up a k8s cluster and builds+deploy kubecf
   # https://github.com/SUSE/catapult/wiki/Build-and-run-SCF#build-and-run-kubecf
   make kubeconfig scf
 
+
+  # TODO: Write the name of the cluster on an S3 text file so it can be shared
+  # with the next jobs (to be able to connect to it)
+
 test_args: &test_args
-- -xce
+- -ce
```

Comment:
This should have never been enabled. We tend to copy and paste this everywhere which could easily lead in leaked credentials. Not sure if there is a way to prevent this once and for all. Maybe this can help? https://concourse-ci.org/creds-redacting.html (although I'm not sure if it's gonna work for a big json file). The last sentence says it all :D `Think of this as an airbag, not a seatbelt!`

Reply:
It's a bit troubling that it's everywhere in this pipeline. Could we please `s/-xce/ce`?

Reply:
agreed , done